Rakefile

```diff
@@ -54,7 +54,7 @@ namespace :server do
       puts "\r\e[0KStopping ..."
       Rake::Task['server:down'].execute
     end
-    system "bundle exec rerun 'rackup -p $PORT -E $RACK_ENV'"
+    system "bundle exec rerun 'rackup -p $PORT -E $RACK_ENV -o 0.0.0.0'"
```

Comment:
I don't know. I haven't tried it on a local environment. I also don't have enough experience with ruby to know if there is a better way to do this without changing the Rakefile.